compiler/onecc-docker/onecc-docker

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python3
+
+# Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import json
+import os
+import requests
+import argparse
```

Comment:
This `argparse` has been added for https://github.com/Samsung/ONE/pull/9737/files#r974949214. Is it going to be used?

Reply:
We haven't started the task on "argparse" yet.
I am trying to use it according to @jyoungyun's  opinion.
I think I can give you a detailed answer after the meeting tomorrow.

Reply:
Could you add token argument for in-house developers?

Reply:
yes let's try that.